app/command.go

```diff
@@ -191,6 +191,100 @@ func (a *App) ExecuteCommand(args *model.CommandArgs) (*model.CommandResponse, *
 	return nil, model.NewAppError("command", "api.command.execute_command.not_found.app_error", map[string]interface{}{"Trigger": trigger}, "", http.StatusNotFound)
 }
 
+type mentionMapItem struct {
```

Comment:
nit: even though we'd have to duplicate, I'd suggest just inlining this definition into each of `mentionsToTeamMembers` and `mentionsToPublicChannels` as a very-internal implementation detail. It's unlikely anyone else in `app` will need this.

Reply:
I agree, thanks for the catch!

Reply:
Done in https://github.com/mattermost/mattermost-server/pull/13762/commits/552c67e6d459e98b646fd149c792a2bb0eb7486d.